examples/02_meta-analyses/10_plot_cbmr.py

```diff
@@ -0,0 +1,362 @@
+# set up group columns: diagnosis & drug_status
+n_rows = dset.annotations.shape[0]
+dset.annotations["diagnosis"] = [
+    "schizophrenia" if i % 2 == 0 else "depression" for i in range(n_rows)
+]
+dset.annotations["drug_status"] = ["Yes" if i % 2 == 0 else "No" for i in range(n_rows)]
+dset.annotations["drug_status"] = (
+    dset.annotations["drug_status"].sample(frac=1).reset_index(drop=True)
+)  # random shuffle drug_status column
+# set up continuous moderators: sample sizes & avg_age
+dset.annotations["sample_sizes"] = [dset.metadata.sample_sizes[i][0] for i in range(n_rows)]
+dset.annotations["avg_age"] = np.arange(n_rows)
+# set up categorical moderators: schizophrenia_subtype (as not enough data to be interpreted
+# as groups)
+dset.annotations["schizophrenia_subtype"] = ["type1", "type2", "type3", "type4", "type5"] * int(
+    n_rows / 5
+)
+dset.annotations["schizophrenia_subtype"] = (
+    dset.annotations["schizophrenia_subtype"].sample(frac=1).reset_index(drop=True)
+)  # random shuffle drug_status column
+
+###############################################################################
+# Estimation of group-specific spatial intensity functions
+# -----------------------------------------------------------------------------
+# Unlike kernel-based CBMR methods (e.g. ALE, MKDA and SDM), CBMR provides a
+# generative regression model that estimates a smooth intensity function and
+# can have study-level moderators. It's developed with a spatial model to
+# induce a smooth response and model the entire image jointly, and fitted with
+# different variants of statistical distributions (Poisson, Negative Binomial
+# (NB) or Clustered NB model) to find the most accurate but parsimonious model.
+#
+# CBMR framework can generate estimation of group-specific spatial internsity
+# functions for multiple groups simultaneously, with different group-specific
+# spatial regression coefficients.
+#
+# CBMR framework can also consider the effects of study-level moderators
+# (e.g. sample size, year of publication) by estimating regression coefficients
+# of moderators (shared by all groups). Note that moderators can only have global
+# effects instead of localized effects within CBMR framework. In the scenario
+# that there're multiple subgroups within a group, while one or more of them don't
+# have enough number of studies to be inferred as a separate group, CBMR can
+# interpret them as categorical study-level moderators.
+from nimare.meta.cbmr import CBMREstimator
+
+dset = StandardizeField(fields=["sample_sizes", "avg_age"]).transform(dset)
+
+cbmr = CBMREstimator(
+    group_categories=["diagnosis", "drug_status"],
+    moderators=[
+        "standardized_sample_sizes",
+        "standardized_avg_age",
+        "schizophrenia_subtype:reference=type1",
+    ],
+    spline_spacing=100,  # a reasonable choice is 10, 100 is for speed
+    model=models.PoissonEstimator,
+    penalty=False,
+    lr=1e-1,
+    tol=1e3,
+    device="cpu",
```

Comment:
You might want to comment on the "cpu" line. i.e. that if you have  GPU you can run on this on GPU, etc...

You may also want to comment on the other parameters, because as a naiive end use I wonder: why are the defaults for those parameters not good enough? How did you come they come up w/ those parameters?

Reply:
Thanks for the suggestions! I've added an explanation that the current choice of parameters are only for speed